pep-0570.rst

```diff
@@ -385,6 +385,305 @@ as a rule is not LL(1). This is the reason the rule needs to be included in
 the existing typedargslist and varargslist (in the same way keyword-only
 arguments were introduced).
 
+=================
+How to teach this
+=================
+
+This PEP recommends adding a new subsection to the Python documentation,
+in the section "More on Defining Functions", where the rest of the
+argument types are discussed. The following paragraphs serve as a draft
+for the mentioned additions that will serve to explain what positional-only
+parameters are and some scenarios where they can be useful. It does not intend
+to be an exhaustive document nor the final version that will be incorporated
+in the documentation.
+
+-------------------------------------
+How to use positional-only parameters
+-------------------------------------
+
+You can mark a set of parameters in a function to be positional-only by placing
+a "/" marker after them ::
+
+    def process_spams(spam1, spam2, /, complementary_spam):
+        ....
+
+In this example, ``spam1`` and ``spam2`` are positional-only while
```

Comment:
Agreed. A bit too much spam which adds to the cognitive load. More distinct names would be better as this will likely become end users docs at some point. I'll give some thought to a real world example.